src/core/ext/transport/chttp2/transport/hpack_parser.cc

```diff
@@ -40,6 +40,14 @@
 #include "src/core/lib/surface/validate_metadata.h"
 #include "src/core/lib/transport/http2_errors.h"
 
+#if __cplusplus > 201103L
```

Comment:
do we need this? Aren't we already requiring C++11?

Reply:
`>` not `>=`

Reply:
misread. gotcha